framework/jaif-inference-scripts/insert-annotations.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
```

Comment:
The specified JAIFs all get combined into a single "scene", so there's no need to associate specific index files with specific source files.